news/2024/2024-09-11-osumania-4k-world-cup-2024-quarterfinals-recap.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "osu!mania 4K World Cup 2024: Quarterfinals Recap"
+date: 2024-09-11 21:00:00 +0000
+---
+
+Honey, it's time to consume your weekly dose of World Cup recaps...
```

Comment:
i don't think this tone is quite appropriate

Reply:
c32debfb52f88cf540adabaa99a5176055202f15 changed to something more appropriate